### DISCUSSION TITLE
Release Schedule

### DISCUSSION THREAD

**Discussion by alexanderson1993** [Thorium-Sim/thorium-nova#19] [Ideas] [answered]
I that we can release a weekly Alpha build automatically. This would involved creating a bot that automatically does a merge from `develop` to `main` once a week, which would trigger the automated release process.

The great thing about this is it would only create a release if there were commits during the week. If folks adhere to the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) spec, it automatically generates release notes too, which is nice.

Any objections to me setting that up?

**Comment by Xansta** [comment, marked as answer]
An automated release sounds good to me. This also facilitates continuous integration. From the releases generated, we can select the ones that move on to integration testing, user acceptance testing, full release, etc.

No objection

**Reply by alexanderson1993** [reply]
We'll need to figure out some way to have multiple release tracks. I think during the alpha phase, we'll just have the weekly release. Once we hit beta (feature complete) and 1.0, we'll change the release cadence and make it so people can opt-in to pre-release builds if they want to live on the edge.
